pandas/core/computation/eval.py

```diff
@@ -148,7 +147,7 @@ def _check_for_locals(expr, stack_level, parser):
 
 def eval(expr, parser='pandas', engine=None, truediv=True,
          local_dict=None, global_dict=None, resolvers=(), level=0,
-         target=None, inplace=None):
+         target=None, inplace=False):
```

Comment:
@chris-b1 : True, but "with a `target`" is not well-defined either.  As you can see by my documentation change on `target` in `pandas.eval`, you first need to ensure that item assignment with strings is supported, but how do you check that without modifying the object?  That's something I imagined would create unnecessary overhead.  Also, you should then check if there is item assignment in the first place.

That's why I decided to allow the user to pass in `inplace=True` even if there is no real target.  I will in fact document that behavior just in case.